src/MediaCodec.cpp

```diff
@@ -33,6 +33,7 @@ int CJNIMediaCodec::BUFFER_FLAG_END_OF_STREAM(4);
 int CJNIMediaCodec::BUFFER_FLAG_SYNC_FRAME(1);
 int CJNIMediaCodec::CONFIGURE_FLAG_ENCODE(1);
 int CJNIMediaCodec::CONFIGURE_FLAG_DECODE(0);
+int CJNIMediaCodec::CRYPTO_MODE_AES_CBC(2);
 int CJNIMediaCodec::CRYPTO_MODE_AES_CTR(1);
```

Comment:
My understanding is there should be a matching entry in https://github.com/xbmc/libandroidjni/blob/1580f21d73f17aa52d9b07282db53192c25c1d8e/src/MediaCodec.cpp#L46

Be aware, this was introduced in SDK 24, so you will need a GetSDKVersion around it, as kodi currently ships out with minimum SDK of 21.

Reply:
SDK check will be done in DVD*****CodecAndroidMediaCodec, I'll add here though anyway as at the very least it can be useful to document it.